scripts/rspec/rspec-templates/CSharpTestTemplate.cs

```diff
@@ -18,10 +18,9 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-extern alias csharp;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using csharp::SonarAnalyzer.Rules.CSharp;
 using SonarAnalyzer.UnitTest.TestFramework;
+using CSharp = SonarAnalyzer.Rules.CSharp;
```

Comment:
Getting ready for VB
```suggestion
using CS = SonarAnalyzer.Rules.CSharp;
```